packages/vats/test/vpurse.test.js

```diff
@@ -176,6 +177,86 @@ test('makeVirtualPurse', async t => {
     .then(checkWithdrawal);
 });
 
+// TODO Once https://github.com/Agoric/agoric-sdk/issues/9407 is fixed,
+// This test should replace the similar one above.
+test.failing('makeVirtualPurse with optAmountShape pattern', async t => {
```

Comment:
Is there anything else to do besides removing the `.failing` once fixed?

Reply:
The failing test is a trivial variant of the normal test above it, duplicated only to test this additional aspect. https://github.com/Agoric/agoric-sdk/pull/9863 deduplicates it, but has the one test keep the test of this additional aspect.

Resolving.